### PR TITLE
integrate latest Godeps

### PR DESCRIPTION
**Pull request by ElvinEfendi**
This is the first step to move the project out of Packserv. It simplifies test and build scripts and integrates the latest Godeps. Once this is merged I'll integrate Circle CI, package the software as `deb` and push to Packagecloud. As a final step I'll adjust the relevant recipe to install this app from Packagecloud using `apt-get`.

r: @airhorns 
@Shopify/traffic

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package main
 
-const VERSION = "1.0"
+const VERSION = "1.1"
```
Comment (drdee) [inline version.go]: Why bumping the version, this creates a mismatch with the upstream version.

**Comment by drdee** [comment]
@airhorns is on sabbatical. LGTM

**Comment by jduff** [comment]
Could you instead merge the upstream? Looking at this many of our changes could be pushed upstream and we could move off of our fork even.

**Comment by drdee** [comment]
I think our fork is way too old to upstream, upstream is at v2 and has even renamed the project IIRC.

**Comment by ElvinEfendi** [comment]
rel: https://github.com/Shopify/traffic/issues/858

**Pull request merged**